src/main.ts

```diff
@@ -108,3 +98,35 @@ function resolveVersionInput(): string {
 
   return version;
 }
+
+export async function printEnvDetailsAndSetOutput() {
+  core.startGroup('Environment details');
+
+  const promises = ['node', 'npm', 'yarn'].map(async tool => {
+    const output = await getToolVersion(tool, ['--version']);
+
+    core.setOutput(`${tool}-version`, output);
```

Comment:
node-version set on previous steps could you please remove it from there. Besides, could you please logs for versions ?

Reply:
I don't fully understand your comment. Please check those changes https://github.com/actions/setup-node/pull/368/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dL43-L53

Reply:
Sorry, I're right. They were removed. Could you please add core.info to print versions ?

Reply:
Added. Did you mean this? Should I keep that `core.setOutput(`${tool}-version`, output);`?

Reply:
According to the issue it's only about logging. I think for now we can remove output If we get separate issue about output we'll add it. Thank you.

Reply:
the previous version had `node-version` in the output, should we keep it?

Reply:
For node-version we should keep it.

Reply:
Done